core/backend/src/GeoCoordConfig.ts

```diff
@@ -35,8 +35,9 @@ export class GeoCoordConfig {
   };
 
   private static addGcsWorkspace(dbProps: GcsDbProps) {
+    const dontLoad = true; // ###TODO remove me.
     // override to disable loading GCS data from workspaces
-    if (IModelHost.appWorkspace.settings.getBoolean(GeoCoordConfig.settingName.disableWorkspaces, false))
+    if (dontLoad || IModelHost.appWorkspace.settings.getBoolean(GeoCoordConfig.settingName.disableWorkspaces, false))
```

Comment:
Revert this before merging. On my machine the core-backend tests pause for 1-2 minutes trying and eventually failing to connect to the GCS workspace container.